notes/returned-mail-received.vbs

```diff
@@ -88,19 +99,34 @@ DO
 		Else
 			IF Cdate(date_received) > cdate(date) = TRUE THEN err_msg = err_msg & vbnewline & "You must enter an actual date that is not in the future and is in the footer month that you are working in."
 		End If
-		IF ADDR_actions = "Select One:" THEN err_msg = err_msg & vbCr & "Please chose an action for the returned mail."
-    	IF worker_signature = "" THEN err_msg = err_msg & vbCr & "Please sign your case note."
-    	IF err_msg <> "" THEN MsgBox "*** NOTICE!!! ***" & vbCr & err_msg & vbCr & vbCr & "Please resolve for the script to continue."
-    LOOP UNTIL err_msg = ""
-    CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
-LOOP UNTIL are_we_passworded_out = false					'loops until user passwords back in
+		IF ADDR_actions = "Select:" THEN err_msg = err_msg & vbCr & "Please chose an action for the returned mail."
+		IF client_contact_dropdown = "Select:" THEN err_msg = err_msg & vbCr & "Please chose an action for the client contact."
+		IF worker_signature = "" THEN err_msg = err_msg & vbCr & "Please sign your case note."
+		IF ButtonPressed = CASH_POLI_TEMP_button THEN CALL view_poli_temp("02", "08", "011", "") 'TE02.08.011' RETURNED MAIL PROCESSING - CASH
+		IF ButtonPressed = SNAP_POLI_TEMP_button THEN CALL view_poli_temp("02", "08", "012", "") 'TE02.08.012' RETURNED MAIL PROCESSING - SNAP
+		IF ButtonPressed = one_source_button THEN run "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe https://www.dhs.state.mn.us/cs/login/login.htm"
+		IF ButtonPressed = one_source_button or ButtonPressed = SNAP_POLI_TEMP_button or ButtonPressed = CASH_POLI_TEMP_button THEN
+			err_msg = "LOOP"
+		Else                                                'If the instructions button was NOT pressed, we want to display the error message if it exists.
+			IF err_msg <> "" THEN MsgBox "*** NOTICE!***" & vbNewLine & err_msg & vbNewLine
+		End If
+	Loop until err_msg = ""
+	call back_to_self ' this is for if the worker has used the POLI/TEMP navigation'
+	CALL check_for_password(are_we_passworded_out)			'function that checks to ensure that the user has not passworded out of MAXIS, allows user to password back into MAXIS
+LOOP UNTIL are_we_passworded_out = False					'loops until user passwords back in
 
 'setting the footer month to make the updates in'
 CALL convert_date_into_MAXIS_footer_month(date_received, MAXIS_footer_month, MAXIS_footer_year)
 MAXIS_footer_month_confirmation
 
 CALL navigate_to_MAXIS_screen_review_PRIV("STAT", "ADDR", is_this_priv)
-IF is_this_priv = TRUE THEN script_end_procedure("This case is privileged, the script will now end.")
+IF is_this_priv = TRUE THEN script_end_procedure("This case is privileged, please request access to the case and run the script again.")
+
+EMReadScreen error_message, 75, 24, 2 'reading for messages that might be missed if they are not inhibiting'
```

Comment:
I'm not sure what this is for. I see this variable reused on line 388 and then added to the script run lowdown. If you are trying to pass this through - hitting line 388 will overwrite this information and we won't see it. Also using the same variable will make it hard to know which information we are getting in the script run lowdown.

Reply:
I am still not sure how this variable is being handled.

Reply:
removed